bin/cml.js

```diff
@@ -89,13 +107,30 @@ const handleError = (message, error) => {
           description: 'Maximum log level',
           choices: ['error', 'warn', 'info', 'debug'],
           default: 'info'
+        },
+        driver: {
```

Comment:
yargs have a "global option" for these?

for contrast: 
![image](https://user-images.githubusercontent.com/1105843/185990148-e566fe9a-9353-4819-9dbf-b61404679e97.png)
vs
![image](https://user-images.githubusercontent.com/1105843/185990244-f1a21755-cc80-4a35-83aa-6367d62949e1.png)

Reply:
Not that I'm aware of, but it would be nice. 😋

Reply:
If you mean that `yargs` must have a way of displaying global options separately from local options, I haven't found it yet. 🤔

Reply:
Resolved with https://github.com/iterative/cml/pull/1073/commits/a9fac4f6be7313fb0e3d20d0e5ca1b2e63844633